.github/workflows/icon-review.yml

```
@@ -62,30 +62,36 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           script: |
+            const iconsCount = ${{ env.svg_files_count }}
```

```suggestion
            const iconsCount = "${{ env.svg_files_count }}"
```

This is what it was doing before, worth a short.

---

Fixed